package.json

```diff
@@ -70,7 +70,7 @@
     "resolve": "^1.3.3"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "^4.5 || 6.* || >= 7.* || >= 8.*"
```

Comment:
I don't think this change is required, is it? Specifically `>= 7.*` already includes 8. What am I missing?